Server/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+const Camps = require('../database-mongodb/Camp.js')
+
+app.get('/:campsiteId', (req, res) => {
```

Comment:
@alefranco1 @bastipacudan resource path should have some descriptive noun before the id: `'/campsites/:siteId'`